#!/usr/bin/env python
PACKAGE = "jaco_driver"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("X_Pose", double_t, 0, "X Pose", 0, -1, 1)
gen.add("Y_Pose", double_t, 0, "Y Pose",0, -1, 1)
gen.add("Z_Pose", double_t, 0, "Z Pose", 0, -1, 1)
gen.add("RX_Pose", double_t, 0, "RX Pose", 0, -5, 5)
gen.add("RY_Pose", double_t, 0, "RY Pose", 0,-5, 5)
gen.add("RZ_Pose", double_t, 0, "RZ Pose", 0, -5, 5)

exit(gen.generate(PACKAGE, "JacoPosition", "JacoPosition"))
